CONTRIBUTING.md

```diff
@@ -146,9 +146,9 @@ of binaries in ./build directory.
 
 In order to test locally, you'd need a test setup. Local test setup automation is planned but but not done yet, so currently the environment has to be set up manually.
 
-Test environment  typically consist of 1 ESX and 2  guest VMs running inside of the
-ESX. We also support 1 ESX and 1 guest VM. We require ESX 6.0 and later,
-and a Linux VM running  Docker 1.10+ enabled for  plain text TCP connection, i.e.
+Test environment  typically consist of 1 ESX and 3 guest VMs running inside of the
```

Comment:
local test bed is getting more and more complicated to set up :-(
There should be a way to do basic tests on a single VM. (not related to documentation)